packages/contentful/src/gp2/queries/output.queries.ts

```diff
@@ -69,6 +69,17 @@ export const outputsContentQueryFragment = gql`
         name
       }
     }
+    relatedOutputsCollection(limit: 10) {
```

Comment:
I've noticed that it's possible to add more than 10 related outputs in the hub (typing the title of an output after adding all available suggestions). I wonder if there's a way to block adding only 10 in the UI, it was not mentioned in the ticket though. So not sure it's a big issue.